LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Red Hat, Inc.
```

Comment:
IMO LICENSE should not be synchronized from the template at all. it is a choice that you make at the initial iport of the role, and one can not change it later to conform to some "standard".

Reply:
There was no `LICENSE` file previously - there was only a `COPYING` file.  I added the file `LICENSE` which is just a copy of `COPYING`, to make it consistent with the template.  I did not add a new license or change the existing license.  I note that these repos have only a `COPYING` file and no `LICENSE`: firewall linux-system-roles.github.io postfix selinux test-harness tuned - is this OK?  If so, I'll remove `LICENSE` from this PR, and update the auto-maintenance/sync-template.sh script to skip copying `LICENSE` if `COPYING` already exists.

Reply:
I removed `LICENSE` from the PR.

Reply:
I believe sync-template should just avoid copying LICENSE. It does not sound right to add a license if there is none. The repos should have a license in some form, but if they don't, one should not add one as part of some automated process, easily missed among other changes. By doing this, we would effectively relicense the project.
In other words, adding a license file should be a conscious act of the maintainer, not of an automated script.

Reply:
ok - my latest commit in PR https://github.com/linux-system-roles/auto-maintenance/pull/9 removes `LICENSE`